pkg/util/kubernetes/apiserver/events.go

```diff
@@ -0,0 +1,95 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
+// +build kubeapiserver
+
+package apiserver
+
+//// Covered by test/integration/util/kube_apiserver/events_test.go
+
+import (
+	"context"
+	"io"
+	"strconv"
+	"time"
+
+	log "github.com/cihub/seelog"
+	"github.com/ericchiang/k8s"
+	"github.com/ericchiang/k8s/api/v1"
+)
+
+var eventReadTimeout = 100 * time.Millisecond
```

Comment:
**eventReadTimeout** could be a const ?

Reply:
The benefit of using `var` is to force the type. It seems like something we want to enforce as early as possible.